frontend/src/components/BCNavbar/components/HeaderBar.jsx

```diff
@@ -55,7 +56,7 @@ const HeaderBar = props => {
       >
         <Icon fontSize='default'>{mobileNavbar ? 'close' : 'menu'}</Icon>
       </BCBox>
-    </Toolbar>
+    </Toolbar >
```

Comment:
make sure to run a linting check to catch small fixes like this.

Reply:
yes, corrected and fixed other common lint issues.